From 66a2cc02d631b9b5103467a9c15fccc17faef8bd Mon Sep 17 00:00:00 2001
From: Pavel Raiskup <praiskup@redhat.com>
Date: Fri, 4 Mar 2016 13:33:46 +0100
Subject: [PATCH] man-page-day fixes

This is to not let the man-page-day script report issues against
xz again, even though this has not been accepted upsttream.

TODO: Blacklist this in man-page-day testsuite.
---
 src/xz/message.c  | 8 +++++---
 src/xz/xz.1       | 2 +-
 src/xzdec/xzdec.c | 6 ++++--
 3 files changed, 10 insertions(+), 6 deletions(-)

diff --git a/src/xz/message.c b/src/xz/message.c
index f88c123..f4ebddc 100644
--- a/src/xz/message.c
+++ b/src/xz/message.c
@@ -1091,7 +1091,8 @@ message_help(bool long_help)
 
 	puts(_(
 "  -z, --compress      force compression\n"
-"  -d, --decompress    force decompression\n"
+"  -d, --decompress, --uncompress\n"
+"                      force decompression\n"
 "  -t, --test          test compressed file integrity\n"
 "  -l, --list          list information about .xz files"));
 
@@ -1101,7 +1102,8 @@ message_help(bool long_help)
 	puts(_(
 "  -k, --keep          keep (don't delete) input files\n"
 "  -f, --force         force overwrite of output file and (de)compress links\n"
-"  -c, --stdout        write to standard output and don't delete input files"));
+"  -c, --stdout, --to-stdout\n"
+"                      write to standard output and don't delete input files"));
 
 	if (long_help) {
 		puts(_(
@@ -1157,7 +1159,7 @@ message_help(bool long_help)
 		puts(_( // xgettext:no-c-format
 "      --memlimit-compress=LIMIT\n"
 "      --memlimit-decompress=LIMIT\n"
-"  -M, --memlimit=LIMIT\n"
+"  -M, --memlimit=LIMIT, (old alias --memory=LIMIT)\n"
 "                      set memory usage limit for compression, decompression,\n"
 "                      or both; LIMIT is in bytes, % of RAM, or 0 for defaults"));
 
diff --git a/src/xz/xz.1 b/src/xz/xz.1
index bc5514d..b4406d6 100644
--- a/src/xz/xz.1
+++ b/src/xz/xz.1
@@ -1022,7 +1022,7 @@ See
 for possible ways to specify the
 .IR limit .
 .TP
-\fB\-M\fR \fIlimit\fR, \fB\-\-memlimit=\fIlimit\fR, \fB\-\-memory=\fIlimit
+\fB\-M\fR \fIlimit\fR, \fB\-\-memlimit=\fIlimit\fR, (old alias \fB\-\-memory=\fIlimit\fR)
 This is equivalent to specifying \fB\-\-memlimit\-compress=\fIlimit
 \fB\-\-memlimit\-decompress=\fIlimit\fR.
 .TP
diff --git a/src/xzdec/xzdec.c b/src/xzdec/xzdec.c
index 5cb7530..6cab322 100644
--- a/src/xzdec/xzdec.c
+++ b/src/xzdec/xzdec.c
@@ -64,9 +64,11 @@ help(void)
 "Usage: %s [OPTION]... [FILE]...\n"
 "Decompress files in the ." TOOL_FORMAT " format to standard output.\n"
 "\n"
-"  -d, --decompress   (ignored, only decompression is supported)\n"
+"  -d, --decompress, --uncompress\n"
+"                     (ignored, only decompression is supported)\n"
 "  -k, --keep         (ignored, files are never deleted)\n"
-"  -c, --stdout       (ignored, output is always written to standard output)\n"
+"  -c, --stdout, --to-stdout\""
+"                     (ignored, output is always written to standard output)\n"
 "  -q, --quiet        specify *twice* to suppress errors\n"
 "  -Q, --no-warn      (ignored, the exit status 2 is never used)\n"
 "  -h, --help         display this help and exit\n"
-- 
2.5.0

